minard/views.py

```diff
@@ -104,6 +104,10 @@ def state(run=None):
     if run_state['caen'] is not None:
         caen_state = detector_state.get_caen_state(run_state['caen'])
 
+    #tubii_state = None
+    #if run_state['tubii'] is not None:
```

Comment:
Yes. Its was previously commented out as I had to hardcode a few things as there wasn't real TUBii data in the DB. That should be fixed now but still seems to give an issue that I'd like Eric to take a look at. I submitted the PR anyway so he could take a look.